proto/cosmos/protocolpool/v1/tx.proto

```diff
@@ -0,0 +1,130 @@
+syntax = "proto3";
+package cosmos.protocolpool.v1;
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/protocolpool/types";
+
+import "cosmos/protocolpool/v1/types.proto";
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos_proto/cosmos.proto";
+import "cosmos/msg/v1/msg.proto";
+import "google/protobuf/timestamp.proto";
+
+// Msg defines the pool Msg service.
+service Msg {
+  option (cosmos.msg.v1.service) = true;
+
+  // FundCommunityPool defines a method to allow an account to directly
+  // fund the community pool.
+  rpc FundCommunityPool(MsgFundCommunityPool) returns (MsgFundCommunityPoolResponse);
+
+  // CommunityPoolSpend defines a governance operation for sending tokens from
+  // the community pool in the x/protocolpool module to another account, which
+  // could be the governance module itself. The authority is defined in the
+  // keeper.
+  rpc CommunityPoolSpend(MsgCommunityPoolSpend) returns (MsgCommunityPoolSpendResponse);
+
+  // CreateContinuousFund defines a method to distribute a percentage of funds to an address continuously.
+  // This ContinuousFund can be indefinite or run until a given expiry time.
+  rpc CreateContinuousFund(MsgCreateContinuousFund) returns (MsgCreateContinuousFundResponse);
```

Comment:
we should specify that these funds come from block rewards. and maybe the possibility that if the module address is externally funded for whatever reason.

Reply:
updated